modules/rules_go/0.33.0/MODULE.bazel

```diff
@@ -0,0 +1,23 @@
+module(
+    name = "rules_go",
+    version = "0.33.0",
+    compatibility_level = 0,
+    toolchains_to_register = [
+        "@go_default_sdk//:all",
+    ],
+)
+
+print("WARNING: The rules_go Bazel module is still highly experimental and subject to change at any time. Only use it to try out bzlmod for now.")
+
+bazel_dep(name = "bazel_skylib", version = "1.2.0")
+bazel_dep(name = "platforms", version = "0.0.4")
+
+non_module_dependencies = use_extension("@rules_go//go/private:extensions.bzl", "non_module_dependencies")
+use_repo(
+    non_module_dependencies,
+    "io_bazel_rules_nogo",
+)
+
+go_sdk = use_extension("@rules_go//go:extensions.bzl", "go_sdk")
+go_sdk.download(name = "go_default_sdk", version = "1.18.3")
+use_repo(go_sdk, "go_default_sdk")
```

Comment:
it would be interesting to see what this looks like if the app uses `go_repository` to fetch and use a third-party module, partly just to motivate the bzlmod team to see the problem with `use_repo` having to list them all again